modules/snooker/src/main/ts/ephox/snooker/util/CellUtils.ts

```diff
@@ -1,26 +1,21 @@
 import { Fun } from '@ephox/katamari';
 import { Attr, Css, Element } from '@ephox/sugar';
 
-const getSpan = function (cell: Element, type: 'colspan' | 'rowspan') {
+export const getSpan = function (cell: Element, type: 'colspan' | 'rowspan') {
   return Attr.has(cell, type) && parseInt(Attr.get(cell, type), 10) > 1;
 };
 
-const hasColspan = function (cell: Element) {
+export const hasColspan = function (cell: Element) {
   return getSpan(cell, 'colspan');
 };
 
-const hasRowspan = function (cell: Element) {
+export const hasRowspan = function (cell: Element) {
   return getSpan(cell, 'rowspan');
 };
 
-const getInt = function (element: Element, property: string) {
+export const getInt = function (element: Element, property: string) {
   return parseInt(Css.get(element, property), 10);
 };
 
-export default {
-  hasColspan,
-  hasRowspan,
-  minWidth: Fun.constant(10),
-  minHeight: Fun.constant(10),
-  getInt
-};
\ No newline at end of file
+export const minWidth = Fun.constant(10);
```

Comment:
Just a very minor optimization, but we could save some additional code size here if we make this share the same variable. eg:

```
const ten = Fun.constant(10);
export const minWidth = ten;
export const minHeight = ten;
```